test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/ProxyHttpRequest/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "isProxy":"true"
+    },
+    {
+      "name": "$return",
+      "type": "http",
+      "direction": "out",
+      "isProxy":"true"
```

Comment:
Also, can we just have isProxy on the trigger, not specified again on the output binding?